<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1717</width>
    <height>1035</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="wgtSidebar" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>150</width>
      <height>880</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>83</y>
       <width>131</width>
       <height>321</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pbGrayScale">
        <property name="toolTip">
         <string>Convert to Grayscale image</string>
        </property>
        <property name="text">
         <string>GrayScale</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pbBlur">
          <property name="toolTip">
           <string>no functionality yet</string>
          </property>
          <property name="text">
           <string>Blur</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="sldrBlur">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>59</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TickPosition::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbReset">
        <property name="toolTip">
         <string>Reset changes</string>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPB1">
        <property name="toolTip">
         <string>no functionality yet</string>
        </property>
        <property name="text">
         <string>PB1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPB2">
        <property name="toolTip">
         <string>no functionality yet</string>
        </property>
        <property name="text">
         <string>PB2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbTest">
        <property name="toolTip">
         <string>Quickly load a Giraffe image</string>
        </property>
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbExit">
        <property name="toolTip">
         <string>Exit Application</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="wgtMain" native="true">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>10</y>
      <width>1461</width>
      <height>879</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(131, 148, 150);
</string>
    </property>
    <widget class="QLabel" name="lblmage">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>15</y>
       <width>1441</width>
       <height>851</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
     <property name="text">
      <string>Load an Image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1717</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(98, 160, 234);
selection-background-color: rgb(255, 163, 72);</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actOpenImage"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actOpenImage">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
   </property>
   <property name="text">
    <string>Open Image...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowClose"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
